lib/index.js

```diff
@@ -16,29 +16,24 @@ const MATCH_IMPORTS = /@import\s+(['"])([^,;'"]+)(\1)(\s*,\s*(['"])([^,;'"]+)(\1
 const MATCH_FILES = /(['"])([^,;'"]+)(\1)/g
 
 function getImportsToResolve (original, includePaths) {
-  let extname = path.extname(original)
-  let basename = path.basename(original, extname)
-  let dirname = path.dirname(original)
-  let basedirs = [dirname].concat(includePaths)
+  const extname = path.extname(original)
+  const basename = path.basename(original, extname)
+  const dirname = path.dirname(original)
+  const imports = []
+  const names = [basename]
+  const exts = extname ? [extname] : EXT_PRECEDENCE
 
-  let imports = []
-  let names = [basename]
-  let exts = [extname]
-
-  if (!extname) {
-    exts = EXT_PRECEDENCE
-  }
   if (basename[0] !== '_') {
     names.push('_' + basename)
   }
-
-  for (let i = 0; i < names.length; i++) {
-    for (let j = 0; j < exts.length; j++) {
-      for (let basedir of basedirs) {
-        imports.push(path.join(basedir, names[i] + exts[j]))
-      }
-    }
-  }
+  names.forEach((name) => {
+    exts.forEach((ext) => {
+      imports.push(path.join(dirname, `${name}${ext}`))
+      includePaths.forEach((includePath) => {
+        imports.push(path.resolve(includePath, path.join(dirname, `${name}${ext}`)))
```

Comment:
I think this place should be:

```js
path.resolve(dirname, includePath, `${name}${ext}`)
```

`includePath` should resolved from `dirname`, not the opposite